### ISSUE TITLE
newest version of dapple on npm doesn't work with the solc compiler

### ISSUE THREAD

**Issue by apmilen** [nexusdev/dapple#318]
the newest version of dapple on npm doesn't work with its solc-js compiler. I upgraded the solc npm dependency in my local repo to 0.4.2 and got this output for both `dapple build` and `dapple test`:

```
~/workplace/milenius/rlp-calculator $ dapple-local build
Building...
/bin/sh: solc: command not found
No local solc found. Failing over to JS compiler...
dapple/debug.sol:1:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.4.2;".
contract Debug {
^
Spanning multiple lines.
dapple/test.sol:1:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.4.2;".
import 'dapple/debug.sol';
^
Spanning multiple lines.
/Users/amilenius/workplace/milenius/rlp-calculator/contracts/rlp.sol:1:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.4.2;".
import "dapple/test.sol";
^
Spanning multiple lines.
dapple/dapple_log.sol:1:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.4.2;".
contract DappleLogger {
^
Spanning multiple lines.
dapple/environment.sol:1:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.4.2;".
import "/Users/amilenius/workplace/milenius/rlp-calculator/contracts/rlp.sol";
^
Spanning multiple lines.
dapple/reporter.sol:1:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.4.2;".
import "dapple/dapple_log.sol";
^
Spanning multiple lines.
dapple/script.sol:1:1: Warning: Source file does not specify required compiler version! Consider adding "pragma solidity ^0.4.2;".
import "dapple/environment.sol";
^
Spanning multiple lines.
```

Nothing got built in either case, so it appears to be silently failing somewhere. Any idea about what could be going wrong?

**Comment by mhhf** [comment]
should be fixed with 0.8.1
plz try and report

**Comment by apmilen** [comment]
works great now. thanks
